books/free-programming-books-subjects.md

```diff
@@ -857,6 +857,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Code Simplicity: The Fundamentals of Software](https://www.codesimplicity.com/book.pdf) - Max Kanat-Alexander (PDF)
 * [Data-Oriented Design](https://www.dataorienteddesign.com/dodmain/dodmain.html)
 * [Designing Event-Driven Systems. Concepts and Patterns for Streaming Services with Apache Kafka](https://assets.confluent.io/m/7a91acf41502a75e/original/20180328-EB-Confluent_Designing_Event_Driven_Systems.pdf) - Ben Stopford (PDF)
+* [Designing Web APIs](https://www.pdfdrive.com/designing-web-apis-building-apis-that-developers-love-e176234255.html) -  Brenda Jin, Saurabh Sahni, Amir Shevat (HTML)
```

Comment:
PDFDrive links are not supported. Please provide official and authorized links which users can use to access the resource. Read docs/CONTRIBUTING.md fore more information